simdjson.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
```

Comment:
Why not just use the `JoinPaths` module anyway, for now, so that you properly handle the case where CMAKE_INSTALL_LIBDIR is an absolute path?

Reply:
Mainly because it seems that it doesn't work well on Windows, but yeah it's better than nothing. I'll add it later.